base.pxt

```diff
@@ -801,7 +803,13 @@ WindowProperty:FrameAndDivider,
   <%= win_edge_u_ratio %>,  !- Ratio of Divider-Edge Glass Conductance to Center-Of-Glass Conductance
   <%= frame_divider_absorptance %>,                     !- Divider Solar Absorptance
   0,                       !- Divider Visible Absorptance
-  0.9;                     !- Divider Thermal Hemispherical Emissivity
+  0.9,                     !- Divider Thermal Hemispherical Emissivity
+  0.0,                     !- Outside Reveal Solar Absorptance
+  0.0,                     !- Inside Sill Depth
+  0.0,                     !- Inside Sill Solar Absorptance
+  0.0,                     !- Inside Reveal Depth
+  0.0,                     !- Inside Reveal Solar Absorptance
+  CurtainWall;             !- NFRC Product Type for Assembly Calculations

```

Comment:
Is this really necessary? There isn't a default?

Reply:
The `\minfields` IDD rule increased from 20 => 26 for the `WindowProperty:FrameAndDivider` object, making these 6 inputs now required. All of these values are the IDD defaults, so I could reset them to blank fields.